daprdocs/content/en/js-sdk-docs/js-actors/_index.md

```diff
@@ -49,149 +51,149 @@ export default class ParkingSensorImpl extends AbstractActor implements ParkingS
     // Implementation that updates state that this parking spaces is available.
   }
 
-  async getParkingSpaceUpdate(): Promise<object> {
+  private async getInfo(): Promise<object> {
     // Implementation of requesting an update from the parking space sensor.
   }
 
+  /**
+   * @override
+   */
   async onActivate(): Promise<void> {
     // Initialization logic called by AbstractActor.
   }
 }
 ```
 
 ## Registering Actors
-Initialize and register your actors by using the DaprServer package:
+Initialize and register your actors by using the `DaprServer` package:
 
 ```javascript
 import { DaprServer } from "dapr-server";
 import ParkingSensorImpl from "./ParkingSensorImpl";
 
-async function start() {
-  const server = new DaprServer(`server-host`, `server-port`, `dapr-host`, `dapr-port`);
+const daprHost = "127.0.0.1";
+const daprPort = "50000";
+const serverHost = "127.0.0.1";
+const serverPort = "50001";
 
-  await server.actor.init(); // Let the server know we need actors
-  server.actor.registerActor(ParkingSensorImpl); // Register the actor
-  await server.start(); // Start the server
-}
+const server = new DaprServer(serverHost, serverPort, daprHost, daprPort);
+
+await server.actor.init(); // Let the server know we need actors
+server.actor.registerActor(ParkingSensorImpl); // Register the actor
+await server.start(); // Start the server
+
+// To get the registered actors, you can invoke `getRegisteredActors`:
+const resRegisteredActors = await server.actor.getRegisteredActors();
+console.log(`Registered Actors: ${JSON.stringify(resRegisteredActors)}`);
 ```                                              
 
-## Invoking Actors
-After Actors are registered, use the DaprClient to invoke methods on an actor. The client will call the actor methods defined in the actor interface.
+## Invoking Actor Methods
+After Actors are registered, use the `DaprClient` to invoke methods on an actor. The client will call the actor methods defined in the actor interface.
```

Comment:
Maybe add how we will invoke? e.g. "we will be able to invoke methods by calling a method on the Proxy object of the actor which will translate the call for us". To add some context on the behind-the-scenes